DataFormats/Detectors/TRD/include/DataFormatsTRD/EventRecord.h

```diff
@@ -93,6 +94,9 @@ class EventStorage
   void addTracklets(InteractionRecord& ir, std::vector<Tracklet64>& tracklets);
   void addTracklets(InteractionRecord& ir, std::vector<Tracklet64>::iterator& start, std::vector<Tracklet64>::iterator& end);
   void unpackDataForSending(std::vector<TriggerRecord>& triggers, std::vector<Tracklet64>& tracklets, std::vector<Digit>& digits);
+  void sendData(o2::framework::ProcessingContext& pc);
```

Comment:
Do I understand correctly that you fill the vectors twice, once in the unpackDataForSending and also in sendData?

Reply:
unpackDataForSending is just for debugging, the production version only uses sendData. It was easier to do it that way when i wanted to get access to all of vectors. I renamed it now to simpy unpackData